### PR TITLE
Fix copypasta in async/await Learn article

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/5481. Also fixes some flaws.

### CODE REVIEW

File: files/en-us/learn/javascript/asynchronous/async_await/index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
...
+<p class="summary">More recent additions to the JavaScript language are <a href="/en-US/docs/Web/JavaScript/Reference/Statements/async_function">async functions</a> and the <code><a href="/en-US/docs/Web/JavaScript/Reference/Operators/await">await</a></code> keyword, part of the so-called ECMAScript 2017 JavaScript edition. These features basically act as syntactic sugar on top of promises, making asynchronous code easier to write and to read afterwards. They make async code look more like old-school synchronous code, so they're well worth learning. This article gives you what you need to know.</p>
```
Comment (hamishwillee) [inline files/en-us/learn/javascript/asynchronous/async_await/index.html:19]: This is good, but can we/should we also get rid of "so called"?

**Review by hamishwillee — APPROVED** [review]

File: files/en-us/learn/javascript/asynchronous/async_await/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/learn/javascript/asynchronous/async_await/index.html:19]: Also, what is "copypasta" in heading :-) ?

Comment (sideshowbarker) [reply files/en-us/learn/javascript/asynchronous/async_await/index.html]: > This is good, but can we/should we also get rid of "so called"?

Yup — dropped now

Comment (sideshowbarker) [reply files/en-us/learn/javascript/asynchronous/async_await/index.html]: > Also, what is "copypasta" in heading :-) ?

Shorthand for “copy/paste error” — referring to the fact that the *“To understand promises and how to use them”* text from the corresponding Learn page for Promises got copy/pasted into this page unchanged.

**Pull request merged**